libs/shared/src/entities/change/change.interface.ts

```diff
@@ -1,6 +1,7 @@
 export enum ChangeEntityTypeEnum {
-  NOTIFICATION_TEMPLATE = 'NotificationTemplate',
+  FEED = 'FEED',
```

Comment:
```suggestion
  FEED = 'Feed',
```

Reply:
I was meant to write it like your suggestion. Don't know what happened. 🤦🏻